.github/workflows/main.yml

```diff
@@ -37,6 +37,8 @@ jobs:
           package-lock.json
           server/package-lock.json
           client/package-lock.json
+          server/tree-sitter-bash.info
```

Comment:
Do they ever change in this workflow? Also, the npm cache shouldn't depend on them.

Reply:
No, they don't change in this workflow.

I am trying to cache the result of this step so it is executed only if these files are modified, since generating wasm files takes time. However this workflow has run many times now and it does not seem to work. Not sure whether it is because the change is not yet included on the main branch, or because I misunderstand [the purpose of the option](https://github.com/actions/setup-node?tab=readme-ov-file#usage).

Reply:
I moved the cache to a separate task. It works now.